packages-ts/gauntlet-terra-contracts/src/lib/middlewares.ts

```diff
@@ -1,5 +1,6 @@
-import { TerraCommand, AddressBook, logger } from '@chainlink/gauntlet-terra'
+import { TerraCommand, logger } from '@chainlink/gauntlet-terra'
 import { Middleware, Next } from '@chainlink/gauntlet-core'
+import { AddressBook } from '@chainlink/gauntlet-core/dist/commands/addressBook'
```

Comment:
Let's make the change on `gauntlet-core` first, make the release, and then update `gauntlet-terra`